dev/TEAL_opcodes.md

```diff
@@ -1046,339 +959,352 @@ params: Before v4, Txn.ForeignAssets offset. Since v4, Txn.ForeignAssets offset
 | 8 | AppAddress | []byte | Address for which this application has authority |
 
 
-params: Txn.ForeignApps offset or an app id that appears in Txn.ForeignApps. Return: did_exist flag (1 if the application existed and 0 otherwise), value.
+params: Txn.ForeignApps offset or an _available_ app id. Return: did_exist flag (1 if the application existed and 0 otherwise), value.
+
+## acct_params_get f
```

Comment:
Minor: `app_params_get` recall the allowed fields while `acct_params_get` does not.